server.ts

```diff
@@ -21,11 +21,9 @@ const crypto_algorithm = "aes-192-cbc";
 //PLACEHOLDER VALUES FOR CRYPTO. DO NOT USE FOR PRODUCTION. Replace "researcherpassword" with researcher"s password.
 const private_key_example = crypto.scryptSync("researcherpassword", "salt", 24);
 const iv_example = crypto.randomBytes(16);
-const csrfProtection = csrf()
-
 
 app.use(cors());
-app.use(cookieParser());
+//app.use(cookieParser()); // Not using cookies rn, so just turned it off for now
```

Comment:
So, the code previously was using both express-session and cookie-parser, but since "1.5.0, the cookie-parser middleware no longer needs to be used for this module to work."  So, still using server-side cookies here. Just took out redundant package cookie-parser.

On the question of req.session.id, express-session just assign unique id to incoming session. Linking the express-session docs for reference.

http://expressjs.com/en/resources/middleware/session.html

Reply:
Thanks!